integration-tests/initiateDataRequest.spec.ts

```diff
@@ -1,16 +1,49 @@
-import {
-  getMatchingLogEvents,
-  extractRequestIDFromEventMessage,
-  waitForLogStreamContainingEvent
-} from './utils/aws/cloudWatchGetLogs'
+import { getCloudWatchLogEventsGroupByMessagePattern } from './utils/aws/cloudWatchGetLogs'
 import { createZendeskTicket } from './utils/zendesk/createZendeskTicket'
 import { approveZendeskTicket } from './utils/zendesk/approveZendeskTicket'
 import { deleteZendeskTicket } from './utils/zendesk/deleteZendeskTicket'
 import { invalidRequestData, validRequestData } from './constants/requestData'
+import {
+  INITIATE_DATA_REQUEST_LAMBDA_LOG_GROUP,
```

Comment:
We should probably use env variables for the log groups as suggested by AndyM? I've changed to that in my local branch.